plugin/completion.py

```diff
@@ -89,8 +89,12 @@ def epilogue(self, item: Dict[str, Any]) -> None:
             self.view.run_command("lsp_apply_document_edit", {'changes': edits})
         command = item.get("command")
         if command:
```

Comment:
Adding a comment saying that this was added to fix a vs code specific implementation would be nice for future reference.

Ideally this should be fixed in the language server, because it editor specific and nowhere in the spec does it say that editors should handle a editor.action.triggerSuggest command  🙂

Reply:
> Ideally this should be fixed in the language server,

Don't think it's possible to do otherwise with LSP.